### PR TITLE
add pkg_latest_version test

### PR DESCRIPTION
**Pull request by frogunder**
### What issues does this PR fix or reference?

#43535

### CODE REVIEW

**Comment by Ch3LL** [comment]
re-run ubuntu

**Comment by Ch3LL** [comment]
re-run ubuntu

**Comment by Ch3LL** [comment]
re-run ubuntu

**Comment by rallytime** [comment]
re-run ubuntu

File: tests/integration/modules/test_pkg.py [hunk 1]
```diff
@@ -326,3 +326,24 @@ def test_pkg_upgrade_has_pending_upgrades(self):
...
+
+    @destructiveTest
+    @skipIf(salt.utils.is_windows(), 'minion is windows')
+    def test_pkg_latest_version(self):
+        '''
+        check that pkg.latest_version returns the latest version of the uninstalled package (it does not install the package, just checking the version)
+        '''
+        grains = self.run_function('grains.items')
+        cmd_info = self.run_function('pkg.info_installed', ['htop'])
+        if cmd_info != '':
```
Comment (Ch3LL) [inline tests/integration/modules/test_pkg.py]: I'm assuming you are checking for a `''` return if the package htop is not installed? If that assumption is correct this means this test will fail if the package is not installed as it wont get a value from cmd_htop as it doesn't get into this if statement.

**Review by Ch3LL — COMMENTED** [review]
One more thing: Looks like this will fail on macosx. Can we just add some logic to ensure we just skip the test if its any other OS?

**Review by rallytime — APPROVED** [review]

**Pull request merged**